test/core/tsi/transport_security_test_lib.cc

```diff
@@ -617,7 +617,7 @@ void tsi_test_fixture_init(tsi_test_fixture* fixture) {
   fixture->notified = false;
 }
 
-void tsi_test_fixture_destroy(tsi_test_fixture* fixture) {
+void tsi_test_fixture_destroy(tsi_test_fixture* fixture, bool free_fixture) {
```

Comment:
I'm not sure how this change got back in here.  I don't think the `free_fixture` parameter is needed.